appveyor.yml

```diff
@@ -0,0 +1,72 @@
+# This file was automatically generated by conda-smithy. To update a component of this
```

Comment:
Part of the reason I'm asking is this looks pretty old. For example it sets `CONDA_PY` generally and it is using `bootstrap-obvious-ci-and-miniconda.py` still. Could we try re-rendering with `conda-smithy` version `2.0.0`?